CHANGELOG.md

```diff
@@ -1,13 +1,16 @@
 # Changelog
 
-## [Unreleased]
+<!-- ## [Unreleased] -->
+
+## [2.0.0] - 2018-08-30
 
 Breaking change: updated dependencies to use PostCSS 6.0.
 
-- Updated autoprefixer: `^6.7.6` -> `^7.2.5` ([changelog](https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md))
-- Updated postcss: `^5.2.15` -> `^6.0.16` ([changelog](https://github.com/postcss/postcss/blob/master/CHANGELOG.md))
+- Updated autoprefixer: `^6.7.6` -> `9.1.3` ([changelog](https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md))
+- Updated postcss: `^5.2.15` -> `7.0.2` ([changelog](https://github.com/postcss/postcss/blob/master/CHANGELOG.md))
 - Updated postcss-calc": `^5.3.1` -> `^6.0.1` ([changelog](https://github.com/postcss/postcss-calc/blob/master/CHANGELOG.md))
-- Updated postcss-flexbugs-fixes: `^2.1.0` -> `^3.2.0` ([changelog](https://github.com/luisrudge/postcss-flexbugs-fixes/blob/master/CHANGELOG.md))
+- Updated postcss-flexbugs-fixes: `^2.1.0` -> `4.1.0` ([changelog](https://github.com/luisrudge/postcss-flexbugs-fixes/blob/master/CHANGELOG.md))
+- Updated postcss-discard-comments: `^2.0.4` -> `4.0.0` ([changelog](https://github.com/cssnano/cssnano/blob/master/packages/postcss-discard-comments/CHANGELOG.md))
 - Updated postcss-selector-matches: `^2.0.5` -> `^3.0.1` ([changelog](https://github.com/postcss/postcss-selector-matches/blob/master/CHANGELOG.md))
 - Updated postcss-will-change: `^1.1.0` -> `^2.0.0` ([changelog](https://github.com/postcss/postcss-will-change/blob/master/CHANGELOG.md))
```

Comment:
If you happen to have time, they may need help on this https://github.com/postcss/postcss-will-change/issues/5